### PR TITLE
Robot control components

### PR DESCRIPTION
**Pull request by destogl**
The third part of #101. 

Depends on #121.

### CODE REVIEW

File: robot_control_components/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.5)
+project(robot_control_components)
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
```
[the remark this answers is not in the document]
Comment (Karsten1987) [reply robot_control_components/CMakeLists.txt:6]: I think the C standard doesn't have to be set here as we won't have any C-only files present. Pretty sure it's just part of the package template.

[the remark this answers is not in the document]
Comment (bmagyar) [reply robot_control_components/CMakeLists.txt:6]: Ah ok, I've seen it in plenty of other packages and wasn't sure if it's needed for idl stuff

**Comment by destogl** [comment]
@bmagyar can you assign this to the project ["ros2_control new HW Interfaces design and examples in demo repository"](https://github.com/orgs/ros-controls/projects/1) somehow I do not have access to it...

**Comment by Karsten1987** [comment]
closing this in favor of #127 
@destogl please feel free to re-open this if I've been mistaken.

**Comment by destogl** [comment]
This is actually the third part of #80. Because of a lot of changes in #121 and #127 I will reopen a new one as soon as the latter is merged.

**Pull request closed without merging**